docs/src/modules/components/overview/charts/mainDemo/DownloadDemo.tsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import { LineChartPro } from '@mui/x-charts-pro/LineChartPro';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import Stack from '@mui/material/Stack';
+// Import data
+import dataMaterial from '../data/@mui-material.json';
+import dataCharts from '../data/@mui-x-charts.json';
+import dataDataGrid from '../data/@mui-x-data-grid.json';
+import dataPickers from '../data/@mui-x-date-pickers.json';
+
+type Versions = '5' | '6' | '7' | '8';
+
+type DataItem = {
+  date: Date;
+} & Partial<Record<Versions, number | undefined | null>> &
+  Partial<Record<`${Versions}_percent`, number | undefined | null>>;
+
+const addRelativeValue = (versions: Versions[]) => (item: DataItem) => {
+  const sum = versions.reduce((a, version: Versions) => a + (item[version] ?? 0), 0);
+
+  for (const version of versions) {
+    if (version in item) {
+      item[`${version}_percent`] = (100 * (item[version] ?? 0)) / sum;
+    } else {
+      item[version] = null;
+      item[`${version}_percent`] = null;
+    }
+  }
+  return item;
+};
+
+const versions: Record<string, Versions[]> = {
+  '@mui/material': ['5', '6', '7'],
+  '@mui/x-data-grid': ['5', '6', '7', '8'],
+  '@mui/x-date-pickers': ['5', '6', '7', '8'],
+  '@mui/x-charts': ['6', '7', '8'],
+};
```

Comment:
would be nice to use the same colors for the same versions, 

eg: on datagrid, v8 is light blue, on chart v8 is red.